.github/ISSUE_TEMPLATE/2_bug_report_macos.yml

```diff
@@ -0,0 +1,71 @@
+name: "Report a macOS Issue"
+description: "Create a report for a runtime related macOS Issue"
+labels: ["os:mac"]
+body:
+- type: textarea
+  attributes:
+      label: "Important Information"
+      description: >
+          - mpv bersion
+           - macOS bersion
+           - Source of the mpv binary or bundle
+           - If known which version of mpv introduced the problem
+           - Possible screenshot or video of visual glitches
+
+          If you're not using git master or the latest release, update.
+          Releases are listed [here](https://github.com/mpv-player/mpv/releases).
+      value: |-
+          - mpv version:
+          - macOS version:
+          - Source:
+          - Introduced in version:
+  validations:
+      required: true
+- type: textarea
+  attributes:
+      label: "Reproduction Steps"
+      description: >
```

Comment:
I would add a not to test the latest stable version and if possible latest HEAD. We have a lot of reports from old mpv versions and generally we support only the last one.

Reply:
did you have anything other on your mind or is something wrong with the current one?

> If you're not using git master or the latest release, update.
> Releases are listed [here](https://github.com/mpv-player/mpv/releases).